src/definitions/modules/progress.js

```diff
@@ -425,18 +425,13 @@ $.fn.progress = function(parameters) {
               var lastNonZeroIndex = -1;
               var percents = values.map(function(value, index) {
                 var $bar = $($bars[index]);
-                if (value === 0) {
```

Comment:
Your change won't work for your new multiple bars. Every 0% bar within a multiple bar would have a default width then.
A more easier fix would be to check for the `$bars` array length instead and leave the old code as it was so far
```javascript
if (value === 0 && $bars.length>0) {
  $bar.css('display', 'none');
} else {
if (firstNonZeroIndex == -1) {
...
```
![image](https://user-images.githubusercontent.com/18379884/55611448-d3f7a580-5785-11e9-96b0-ffb922a2f365.png)

Reply:
Oops.

That fix can not address when every bars are 0% for multiple progress bars: empty progress bar is not displayed.

Reply:
This works for me:
```javascript
...
var percentSum = 0, isMultiple = $bars.length>1;
var percents = values.map(function(value, index) {
  percentSum+=value;
  var allZero = (index===$bars.length-1 && percentSum===0);
  var $bar = $($bars[index]);
  if (value === 0 && isMultiple && !allZero) {
    $bar.css('display', 'none');
  } else {
    if(isMultiple && allZero) {
	  $bar.css('background', 'transparent');
    }
    if (firstNonZeroIndex == -1) {
	  firstNonZeroIndex = index;
    }
    lastNonZeroIndex = index;
    $bar.css({
	  display: 'block',
	  width: value + '%'
    });
  }
  ...
```

Reply:
Thanks!! 🙏 
Fixed in https://github.com/fomantic/Fomantic-UI/commit/a6a07ad93a315e9b1cd1d92239b9d358ba3496ac